Trakt/Api/TraktApi.cs

```diff
@@ -118,7 +118,7 @@ public async Task<TraktScrobbleResponse> SendMovieStatusUpdateAsync(Movie movie,
             var movieData = new TraktScrobbleMovie
             {
                 app_date = DateTimeOffset.Now.Date.ToString("yyyy-MM-dd"),
-                app_version = _appHost.ApplicationVersion.ToString(CultureInfo.InvariantCulture),
+                app_version = _appHost.ApplicationVersion.ToString(),
```

Comment:
Why?

Reply:
ApplicationVersionString exists, use that